Assets/_BForBoss/_Core/Scripts/AmmunitionCountViewBehaviour.cs

```diff
@@ -0,0 +1,41 @@
+using Perigon.Weapons;
+using TMPro;
+using UnityEngine;
+
+namespace BForBoss
+{
+    public class AmmunitionCountViewBehaviour : MonoBehaviour
+    {
+        private const string DefaultAmmunitionLabel = "0 / 0";
+        private const string DefaultWeaponName = "Pistol";
+        [SerializeField] private TMP_Text _ammunitionCountLabel = null;
```

Comment:
Huh, I've always written out TextMeshProUGUI, I had no idea you could just use TMP_Text here